ckcp/openshift_dev_setup.sh

```diff
@@ -275,9 +275,19 @@ patches:
         value: $ckcp_route " >>"$ckcp_temp_dir/kustomization.yaml"
 
   echo -n "  - kcp $kcp_version: "
-  kubectl apply -k "$ckcp_temp_dir" >/dev/null 2>&1
+  kubectl apply -k "$ckcp_temp_dir" >/dev/null
   # Check if ckcp pod status is Ready
-  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null
+  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null 2>&1 || error="true"
+  if [ "$error" == "true" ]; then
+    printf "Debug: print out pod state\n"
+    kubectl -n $ns describe pods
```

Comment:
I'm surprised shellcheck did not raise an error about missing `"`. Can you please have a look?

Reply:
I double checked that shellcheck did not raise an error about missing `"`

Reply:
Done.